.github/workflows/main.yaml

```diff
@@ -13,15 +13,12 @@ on:
 jobs:
   lint:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: [ "3.11", "3.7" ]
     steps:
       - uses: actions/checkout@v3
-      - name: Set up Python ${{ matrix.python-version }}
+      - name: Set up Python
         uses: actions/setup-python@v3
         with:
-          python-version: ${{ matrix.python-version }}
+          python-version: 3.11
```

Comment:
Wouldn't it make sense to go directly for 3.12? EoF for 3.12 will be reached later than for 3.11 and it's supported too. Or am I missing something else why @pkalita-lbl prefer not to go for 3.12?

Reply:
I see that apparently it doesn't work out-of-the-box with Python 3.12. Therefore I would consider linting with Python 3.11 good enough for the sake of speed getting the original issue fixed.
I'm resolving this conversation so that @pkalita-lbl can decide for himself which way to go: leave it linting with Python 3.11 or invest some extra effor to get it working with Python 3.12.
Both would be IMHO fine.

Reply:
I had originally chosen 3.11 because that was one of (along with 3.7) the versions it had been using previously. As you already saw it failed with 3.12. It's because for some reason the [tox file explicitly asks for](https://github.com/linkml/prefixmaps/blob/0bc0d5fe22e589914f27e5fb557b558945e5653f/tox.ini#L35) `flake8<5.0.0` for that testenv. I couldn't tell you why that is, but such old versions aren't compatible with Python 3.12. I'll leave it as a task for another day to figure out if we can use a later version of `flake8` (or transition to `ruff`).

Reply:
LGTM